in_toto/matchrule_validators.py

```diff
@@ -35,40 +36,47 @@ def _validate_match_rule(keywords):
     (rule, artifact, path_pattern, as_keyword, target_path_pattern, from_keyword,
         step) = keywords
   else:
-    raise FormatError("Wrong rule format, should be: MATCH (MATERIAL/PRODUCT)"
-      "<path_pattern> [AS <target_path_pattern>] FROM <step>.\n\t"
+    raise securesystemslib.exceptions.FormatError("Wrong rule format,"
+      " should be: MATCH (MATERIAL/PRODUCT)"
+      " <path_pattern> [AS <target_path_pattern>] FROM <step>.\n\t"
       "Got: {}".format(" ".join(keywords)))
 
   if rule != "MATCH" and rule.upper() != "MATCH":
-    raise FormatError("Wrong rule to verify! {}".format(rule))
+    raise securesystemslib.exceptions.FormatError(
+        "Wrong rule to verify! {}".format(rule))
 
   if from_keyword != "FROM" and from_keyword.upper() != "FROM":
-    raise FormatError("FROM should come before step")
+    raise securesystemslib.exceptions.FormatError(
+        "FROM should come before step")
 
   if as_keyword != "AS" and as_keyword.upper() != "AS":
-    raise FormatError("AS should come after the step name")
+    raise securesystemslib.exceptions.FormatError(
+        "AS should come after the step name")
 
   if artifact not in MATERIAL_OR_PRODUCT and \
       artifact.upper() not in MATERIAL_OR_PRODUCT:
-    raise FormatError("Target should be either MATERIAL or PRODUCT!")
+    raise securesystemslib.exceptions.FormatError(
+        "Target should be either MATERIAL or PRODUCT!")
 
 def _validate_generic_rule(keywords):
   """ private helper that verifies the syntax of the other rules """
 
   VALID_OPERATIONS = {'CREATE', 'MODIFY', 'DELETE',}
 
   if not isinstance(keywords, list):
-    raise FormatError("this matching rule is not a list")
+    raise securesystemslib.exceptions.FormatError(
+        "this matching rule is not a list")
```

Comment:
I find it helpful if the error message includes the argument/value that is wrong.  When appropriate, examples of valid values also helps.

The following error message, used later in the code, is good:
```Python
raise securesystemslib.exceptions.FormatError("{} is not a valid rule!".format(rule))
```